src/main/resources/config/liquibase/changelog/20250524100749_changelog.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <changeSet id="20250524100749" author="eylulnc">
+        <createTable tableName="global_notification_setting">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false" foreignKeyName="fk_global_notification_setting_user_id" references="jhi_user(id)"/>
+            </column>
+            <column name="global_notification_type" type="ENUM('NEW_LOGIN', 'NEW_PASSKEY_ADDED', 'VCS_TOKEN_EXPIRED', 'SSH_KEY_EXPIRED')">
```

Comment:
Postgres does not support an enum type. please distinguish between mysql and h2 on the one hand and postgres on the other hand. Postgres would then use varchar. You can see an example in the saved post table.